readme.md

```diff
@@ -519,6 +519,7 @@
 - [Drupal](https://github.com/nirgn975/awesome-drupal#readme) - Extensible PHP-based CMS.
 - [Craft CMS](https://github.com/craftcms/awesome#readme) - Content-first CMS.
 - [Sitecore](https://github.com/MartinMiles/Awesome-Sitecore#readme) - .NET digital marketing platform that combines CMS with tools for managing multiple websites.
+- [Silverstripe CMS](https://github.com/wernerkrauss/awesome-silverstripe-cms#readme) - A rapid development MVC framework that serves as a classic fully fledged CMS or as a headless CMS.
```

Comment:
The description could be more succinct.

Maybe:

```suggestion
- [Silverstripe CMS](https://github.com/wernerkrauss/awesome-silverstripe-cms#readme) - MVC framework that serves as a classic or headless CMS.
```

Reply:
I think it should also be mentioned that it's PHP.

Reply:
Thanks for your input. I took your suggestion.